src/screens/CreatePost/index.js

```diff
@@ -868,10 +876,11 @@ const CreatePost = () => {
             backRef={sheetBackRef}
             goBack={() => navigation.goBack()}
             continueToEdit={() => sheetBackRef.current.close()}
+            isAnonimity={typeUser}
           />
         </View>
       </ScrollView>
-      <Loading visible={loading} />
+      <Loading visible={loading} color={useDynamicColors(typeUser).primary} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Ensure that the `isAnonimity` prop is being used in the component and the `color` prop in the `Loading` component is properly handled.
```diff
-            isAnonimity={typeUser}
+            isAnonymous={typeUser}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->